submissions/zhenyakornilov/js-dom/js/index.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+import museums from "./data.js";
+import toggleDarkClass from "./utils.js";
+
+const themeSwitcher = document.getElementById("switcher");
+const listMenu = document.getElementById("menu");
+const dropdownBtn = document.getElementById("dropdown-button");
+const contentBody = document.getElementById("content");
+
+function renderButtons(museumsObj, menu) {
+  let buttonsHTML = museumsObj
+    .map(({ btnName }) => {
+      return `
+      <li class="nav-item">
+        <button class="btn-text">${btnName}</button>
+      </li>
+      `;
+    })
+    .join("");
+
+  menu.innerHTML = buttonsHTML;
+}
+
+function showMuseumInfo({ target }) {
+  if (target.tagName.toLowerCase() === "button") {
+    const currentContentObject = museums.filter(
```

Comment:
you can try to use spread here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax